packages/performance/src/services/perf_logger.ts

```diff
@@ -83,18 +82,16 @@ interface TraceMetric {
   counters?: Array<{ key: string; value: number }>;
   custom_attributes?: Array<{ key: string; value: string }>;
 }
+
 /* eslint-enble camelcase */
 
-let logger: Logger | undefined;
+let logger: (resource: unknown, resourceType: ResourceType) => void | undefined;
```

Comment:
A better type for resource is NetworkRequest | Trace. Same for serializer resource type.

Reply:
Done!